### DISCUSSION TITLE
What are the commands for chat?

### DISCUSSION THREAD

**Discussion by mcordc** [tjarbo/discord-moodle-bot#163] [Q&A] [answered]
Im kind of a noob, but what are the commands for the bot? Thanks.

**Comment by tjarbo** [comment, marked as answer]
Hi :wave: 
thank you for contacting us via the discussions tab. The bot has no commands on discord like `!moodle check` or something similar, as the bot user is only used to send notifications to discord channels. The fmdb is initially configured via the `.env` file or other environment variables and can be adjusted via the web interface.

Hope I could help you with this. If you have any further questions, just open a new discussion :smiley: 

Best regards,
tjarbo
